<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Not hesaplama otomasyonu</string>
   </property>
   <widget class="QPushButton" name="btn_hesapla">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>HESAPLA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_temizle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>TEMİZLE</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>80</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Öğrenci Ad:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ders Ad:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Vize Not:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Final Not</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>127</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="ln_ad"/>
     </item>
     <item>
      <widget class="QLineEdit" name="ln_dersad"/>
     </item>
     <item>
      <widget class="QLineEdit" name="ln_vizenot"/>
     </item>
     <item>
      <widget class="QLineEdit" name="ln_finalnot"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_kapat">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>KAPAT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_kapat</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
